webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-shared-list.js

```diff
@@ -38,7 +38,7 @@ export class SakaiRubricsSharedList extends RubricsElement {
       <div role="tablist">
       ${repeat(this.rubrics, r => r.id, r => html`
         <div class="rubric-item" id="rubric_item_${r.id}">
-          <sakai-rubric-readonly token="${this.token}" rubric="${JSON.stringify(r)}" @copy-to-site="${this.copyToSite}"></sakai-rubric-readonly>
+          <sakai-rubric-readonly token="${this.token}" rubric="${JSON.stringify(r)}" @pdf-rubric="${this.pdfRubric}" @copy-to-site="${this.copyToSite}"></sakai-rubric-readonly>
```

Comment:
Again, why the need for pdf-rubric handling here?

Reply:
The reason is to catch the event and go to the pdfRubric function, to launch the export.